HW2.js

```diff
@@ -0,0 +1,9 @@
+var numArr = process.argv;
+total = 0;
+
+
+for(pos =2; pos <= numArr.length-1; pos++){
```

Comment:
Don't forget to declare your variable here. i.e. `var pos = 2; pos <= numArr.length - 1; pos++)`

Also, instead of saying `pos <= numArr.length - 1` you could use `pos < numArr.length`. Its a little more readable. It doesn't do anything different, and your way is absolutely fine 😄